source/includes/usage-examples/read-code-examples.rb

```diff
@@ -0,0 +1,45 @@
+require 'bundler/inline'
+
+gemfile do
+  source 'https://rubygems.org'
+  gem 'mongo'
+end
+
+uri = '<connection string>'
+
+Mongo::Client.new(uri) do |client|
+
+  # start-find-one
+  document = collection.find(name: '"<value>"').first
```

Comment:
Question for the technical reviewer: Are the ` and " quotes both required?
```suggestion
  document = collection.find(name: "<value>").first
```

Reply:
Good catch. No, using both is incorrect here -- I recommend using only single quotes.